guides/common/modules/snip_backup-usecase-note.adoc

```diff
@@ -0,0 +1,5 @@
+[NOTE]
+====
+If you use the backup tool to create a new instance of the {ProjectServer}, cloned instances are not supposed to run in parallel in a production environment.
+You must decommission the old instance after restoring the backup.
```

Comment:
```suggestion
If you create a new instance of the {ProjectServer}, decommission the old instance after restoring the backup.
Cloned instances are not supposed to run in parallel in a production environment.
```
The note wasn't very clear to me. Perhaps this would make it clearer?